docs/protocol/protocol-reference.mdx

```diff
@@ -0,0 +1,500 @@
+
+:::tip Finding Addresses
+Use `get_config` RPC method to retrieve supported chains and contract addresses dynamically. This ensures you always have the latest deployment information.
+:::
+
+---
+
+## Asset Identifiers
+
+Common asset identifiers in the protocol (lowercase, consistent across chains).
+
+### Supported Assets
+
+| Identifier | Full Name | Decimals | Type |
+|------------|-----------|----------|------|
+| `"usdc"` | USD Coin | 6 | Stablecoin |
+| `"usdt"` | Tether USD | 6 | Stablecoin |
+| `"dai"` | Dai Stablecoin | 18 | Stablecoin |
+| `"eth"` | Ethereum | 18 | Native/Wrapped |
+| `"weth"` | Wrapped Ethereum | 18 | Wrapped Native |
+| `"wbtc"` | Wrapped Bitcoin | 8 | Wrapped BTC |
+
+**Usage in Methods**:
+```javascript
+// Always use lowercase identifier
+transfer({
+  allocations: [
+    {"asset": "usdc", "amount": "100.0"},  // ✓ Correct
+    {"asset": "USDC", "amount": "100.0"}   // ✗ Wrong
+  ]
+})
+```
+
+**Asset Configuration**:
+```javascript
+// Each asset has different contract addresses per chain
+get_assets() → {
+  "usdc": {
+    "symbol": "usdc",
+    "name": "USD Coin",
+    "decimals": 6,
+    "chains": [
+      {"chain_id": 1, "token_address": "0xA0b86991..."},
+      {"chain_id": 137, "token_address": "0x2791Bca1..."},
+      {"chain_id": 42161, "token_address": "0xaf88d065..."}
+    ]
+  }
+}
+```
+
+:::tip Asset Discovery
+Use `get_assets` RPC method to retrieve the complete list of supported assets with contract addresses for each <Tooltip content="A blockchain network identifier (uint64)">chain</Tooltip>. Asset support may vary by clearnode.
+:::
+
+---
+
+## EIP References
+
+Ethereum Improvement Proposals referenced or used by the protocol.
+
+### EIP-191: Signed Data Standard
+
+**Status**: Not used in on-chain signatures (chain-agnostic design)  
+**Link**: https://eips.ethereum.org/EIPS/eip-191
+
+**Why not used**: Nitrolite signatures are computed over raw <Tooltip content="Cryptographic hash of a channel state, used for signature verification">stateHash</Tooltip> without EIP-191 prefix to maintain compatibility across different EVM chains and potential non-EVM implementations.
+
+### EIP-712: Typed Structured Data Hashing
+
+**Status**: Not currently used  
+**Link**: https://eips.ethereum.org/EIPS/eip-712
+
+**Future consideration**: May be adopted for improved wallet UX in future versions.
+
+### EIP-1271: Contract Signature Validation
+
+**Status**: Supported by <Tooltip content="A smart contract that validates state transitions according to application-specific rules">adjudicators</Tooltip>  
+**Link**: https://eips.ethereum.org/EIPS/eip-1271
+
+**Usage**: Enables smart contract wallets to sign <Tooltip content="A snapshot of the channel at a point in time, including fund allocations and application-specific data">state</Tooltip> updates as <Tooltip content="An entity (identified by a wallet address) that is part of a channel">participants</Tooltip>.
+
+### EIP-20 (ERC-20): Token Standard
```

Comment:
That's ERC, so I'm not sure whether it should be included here

Reply:
[ff21dbaf86eb4b18cf046b88a721fd96a2e559f6](https://github.com/layer-3/docs/pull/69/commits/ff21dbaf86eb4b18cf046b88a721fd96a2e559f6)